src/ontology/mondo-edit.obo

```diff
@@ -617334,6 +617334,19 @@ relationship: has_material_basis_in_germline_mutation_in http://identifiers.org/
 property_value: http://purl.org/dc/terms/creator https://orcid.org/0000-0002-7638-4659
 property_value: IAO:0000233 "https://github.com/monarch-initiative/mondo/issues/9001" xsd:anyURI
 
+[Term]
+id: MONDO:1060120
+name: ACO2-related optic atrophy with or without extraocular features
+def: "Any primary mitochondrial disease in which the cause of the disease is monoallelic or biallelic variants in the ACO2 gene. While optic atrophy is present in most affected cases, ACO2 is a mitochondrial protein and thus features of this disease include abnormal mitochondrial morphology and can affect other organ systems. Extraocular features can include ataxia, spastic paraplegia, CNS abnormalities, neurodevelopmental phenotypes, and retinal degeneration." [PMID:29564393]
+synonym: "ACO2-related optic atrophy with or without extraocular features" EXACT CLINGEN_LABEL [PMID:29564393]
+synonym: "infantile cerebellar-retinal degeneration" RELATED [OMIM:616289]
```

Comment:
the xref should be the clingen group, not OMIM because OMIM did not say this is a synonym of this disease. (same with the synonym below.)